master/buildbot/locks.py

```diff
@@ -320,19 +328,23 @@ class LockAccess(util.ComparableMixin):
 
     @param mode: Mode of accessing the lock.
     @type  mode: A string, either 'counting' or 'exclusive'.
+
+    @param count: How many units does the access occupy
+    @type  count: Integer, not negative, default is 1 for backwards
+                  compatibility
     """
 
-    compare_attrs = ('lockid', 'mode')
+    compare_attrs = ('lockid', 'mode', 'count')
 
-    def __init__(self, lockid, mode, _skipChecks=False):
```

Comment:
I'd like to see _skipChecks removal to be moved to a separate commit. If it causes random test failures in the future, it will be possible to bisect the problem to that commit.